tests/unit/models/transactions/test_did_set.py

```diff
@@ -62,3 +62,25 @@ def test_empty(self):
                 error.exception.args[0],
                 "{'did_set': 'Must have one of `did_document`, `data`, and `uri`.'}",
             )
+
+    def test_all_empty_fields(self):
+        with self.assertRaises(XRPLModelException):
+            DIDSet(account=_ACCOUNT, data="", did_document="", uri="")
+
+    def test_delete_multiple_fields(self):
+        # create a valid DID object
+        tx = DIDSet(
+            account=_ACCOUNT,
+            did_document=_VALID_FIELD,
+            uri=_VALID_FIELD,
+            data=_VALID_FIELD,
+        )
+        self.assertTrue(tx.is_valid())
+
+        # remove the data field from the above DID object
+        tx = DIDSet(
```

Comment:
Do you still want a transaction for each field?

Reply:
I'm only removing the `data` field. I'm not using any other transaction

Reply:
You originally also removed the other fields.

Reply:
yes, that's correct. but now, I'm only empty-ing the `data` field

Reply:
Yes, should there be tests for emptying each field?

Reply:
can you tell me the name of the unit test that you are objecting to? 

`test_empty_data_field` -- a unit test that empties only the `data` field. all the other fields contain valid values.

`test_create_did_object_all_empty_fields` --  a unit test that validates the initial input for the `DIDSet` transaction. This verifies that all the input fields are not empty.


You are correct that rippled has already tested this functionality. Which one of these warrants a change?